sdk/core/core-amqp/package.json

```diff
@@ -35,29 +35,29 @@
   ],
   "scripts": {
     "audit": "node ../../../common/scripts/rush-audit.js && rimraf node_modules package-lock.json && npm i --package-lock-only 2>&1 && npm audit",
-    "build:samples": "echo skipped",
+    "build:samples": "echo Obsolete",
     "build:test": "tsc -p . && rollup -c rollup.test.config.js 2>&1",
```

Comment:
I think it's out of scope for this PR, but I'd love us to consider getting rid of this step. Having separate build steps for the lib and for tests causes a bunch of unnecessary churn in CI since usually we end up running tsc twice

Reply:
Perhaps CI should run `rushx build && rushx unit-test`. cc @praveenkuttappan